src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestUrlBuilder.java

```diff
@@ -293,16 +294,12 @@ public void urlBuilderTests() throws Exception {
               DEFAULT_LANGUAGE)
               .getUrlForPath((String) data[1]),
           (String) data[8]);
-      Assert.assertEquals(
-          c.newInstance((String) data[0], "staging", (Boolean) data[2], (Boolean) data[3],
-              DEFAULT_LANGUAGE)
-              .getUrlForPath((String) data[1]),
-          (String) data[9]);
     }
   }
 
   @Test(groups = {"TestUrlBuilder", "unitTests"})
   public void urlWithLanguageBuilderTests() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00112)